sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/MiscTests.cs

```diff
@@ -5,18 +5,20 @@
 using System.Net;
 using System.Net.Http;
 using System.Threading;
-using Xunit;
+using NUnit.Framework;
 using Payload = Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests.Payloads.TokenIssuanceStart;
+using System.Threading.Tasks;
 
 namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests
 {
 
     /// <summary>Tests the OnTokenIssuanceStart request and response for the csharp object model for version preview_10_01_2021</summary>
     [Obsolete]
```

Comment:
Why are we marking this obsolete rather than just removing the test.

Reply:
The obsolete tags here allows us to use obsolete methods and functions for tests, not obsolete the tests for further use.

Reply:
What's the scenario for [Obsolete] on methods being tested here?

We generally do not permit public method to be marked with [Obsolete] as it's a breaking change for the GA surface.  Those should be tagged as editor non-browsable and left functional for back-compat.   For non-GA members, there's no need to obsolete and we should just remove.